portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase

```diff
@@ -665,4 +665,39 @@ definition {
 		}
 	}
 
+	@description = "Story: LRAC-10860 | Automation ID: LRAC-12835 | Test Summary: Select Attributes for Account "
```

Comment:
Hey @manuferreira, I think this test is related with the [LRAC-10865](https://issues.liferay.com/browse/LRAC-10865)